### ISSUE TITLE
3d printed shoulder part

### ISSUE THREAD

**Issue by jgueldenstein** [bit-bots/bitbots_hardware#6]
diagonal like darwin,

check rules first

**Comment by 16stelter** [comment]
> check rules first

RoboCup 2019 rules state "The robot does not possess a configuration where it is extended longer than 1.5·Htop ." and "The minimum length of the arm, measured from the first joint, is Htop − Hleg − Hhead." as the only restrictions for arm length. A diagonal shoulder part would not conflict with these rules and even allow for longer arms if we wanted.

Another rule that references the arms is "The height of the head Hhead, including the neck, satisfies 0.05 · Htop ≤ Hhead ≤ 0.25 · Htop. Hhead is defined
as the vertical distance from the axis of the first arm joint at the shoulder to the top of the head." Since the position of the first motor in the shoulder does not change, this should not be a problem either.

Tl;dr: We're good to go.
